lib/modules/datasource/sbt-plugin/index.ts

```diff
@@ -99,7 +103,9 @@ export class SbtPluginDatasource extends SbtPackageDatasource {
     const repoRoot = ensureTrailingSlash(registryUrl);
     const searchRoots: string[] = [];
     // Optimize lookup order
-    searchRoots.push(`${repoRoot}${groupIdSplit.join('.')}`);
+    if (!registryUrl.startsWith(MAVEN_REPO)) {
```

Comment:
why this new condition?

Reply:
Because we're sure for Maven Central it's always separated with slashes. My guess is that for other registries it's always separated dots, but not sure.